Sources/BottomSheetCalculator.swift

```diff
@@ -30,68 +30,82 @@ struct BottomSheetCalculator {
         return max(superview.frame.height - makeTargetHeight(), handleHeight)
     }
 
-    /// Calculates bottom and top thresholds for the given target offsets.
+    /// Creates the layout of the BottomSheetView based on the target offsets and the current target offset
+    ///
     /// - Parameters:
-    ///   - contentView: the content view of the bottom sheet
+    ///   - targetOffsets: array containing the different target offsets a BottomSheetView can transition between
+    ///   - currentTargetIndex: index of the current target offset of the BottomSheetView
     ///   - superview: the bottom sheet container view
-    ///   - height: preferred height for the content view
-    static func thresholds(for targetOffsets: [CGFloat], in superview: UIView) -> [CGFloat] {
+    ///   - isDismissable: flag specifying whether the last translation target should dismiss the BottomSheetView
+    static func createTranslationTargets(
+        for targetOffsets: [CGFloat],
+        at currentTargetIndex: Int,
+        in superview: UIView,
+        isDismissible: Bool
+    ) -> [TranslationTarget] {
         guard !targetOffsets.isEmpty else { return [] }
 
+        let minOffset = targetOffsets.last ?? 0
+        let maxOffset = targetOffsets.first ?? 0
         let maxThreshold: CGFloat = 75
-        let targetOffsets = [0] + targetOffsets + [superview.frame.height]
 
-        return zip(targetOffsets.dropFirst(), targetOffsets).map {
-            min(abs(($0 - $1) * 0.25), maxThreshold)
-        }
-    }
+        // Thresholds is how long you need to translate from one translation target to another
+        // If the BottomSheetView is dismissible we want the user to translate a certain amount before transitioning to the dismiss translation target
+        // If not, make it stop at the smallest target height by setting the first threshold to zero.
+        let lowestThreshold = isDismissible ? min((abs(superview.frame.height - maxOffset) * 0.25), maxThreshold) : 0
+        // We add a zero threshold at the end to make the BottomSheetView stop at its biggest height.
+        let highestThreshold: CGFloat = 0
+        // Calculate all the offsets in between
+        let thresholds = [lowestThreshold] + zip(targetOffsets.dropFirst(), targetOffsets).map {
+            min((abs($1 - $0) * 0.25), maxThreshold)
```

Comment:
Should we make this into separate func since it's also used on line 55. Could be even a function within `createTranslationTargets` method: 
```swift
func threshold(_ offsetA: CGFloat, offsetB: CGFloat) -> CGFloat
```

Reply:
Good idea 👍